src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin onTablet {
+  @media (min-width: 744px) {
```

Comment:
It's better to use a variable for the tablet breakpoint value (e.g., $tablet-breakpoint) instead of hardcoding it.